scripts/build-kinto-admin.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+TAG=$(cat kinto/plugins/admin/VERSION)
+SRC_DIR="kinto-admin-${TAG}"
+TARBALL_NAME="${SRC_DIR}.tar.gz"
+
+# download and unzip release
+# TODO: error handling (404 error, can't unzip tarball, etc)
+cd kinto/plugins/admin/
+curl -L https://github.com/Kinto/kinto-admin/archive/refs/tags/v${TAG}.tar.gz --output $TARBALL_NAME
+tar -xf $TARBALL_NAME && rm $TARBALL_NAME
+
+# build kinto-admin bundle
+cd $SRC_DIR
+npm ci
+export SINGLE_SERVER=1
```

Comment:
Instead of setting the server here, I was thinking that this could just be a flag so that in kinto-admin, we have something like:
```js
export const SINGLE_SERVER = process.env.SINGLE_SERVER
  ? document.location.toString().split("/admin/")[0]
  : null;
```

Reply:
Yes I don't think we have a way to know the final URL in advance anyway